tests/phpunit/includes/FakeAMPContextPrimary.php

```diff
@@ -0,0 +1,32 @@
+<?php
+/**
+ * Class Google\Site_Kit\Tests\FakeAMPContextPrimary
+ *
+ * @package   Google\Site_Kit\Tests
+ * @copyright 2020 Google LLC
+ * @license   https://www.apache.org/licenses/LICENSE-2.0 Apache License 2.0
+ * @link      https://sitekit.withgoogle.com
+ */
+
+namespace Google\Site_Kit\Tests;
+
+use Google\Site_Kit\Context;
+
+/**
+ * Class for faking a primary AMP context.
+ */
+class FakeAMPContextPrimary extends Context {
```

Comment:
How about `FakePrimaryAMPContext` for the name instead? I think that's more clear because the class is still a `Context`.